cmd/state/internal/cmdtree/bundles.go

```diff
@@ -45,7 +45,7 @@ func newBundlesCommand(prime *primer.Values) *captain.Command {
 func newBundleInstallCommand(prime *primer.Values) *captain.Command {
 	runner := packages.NewInstall(prime)
 
-	params := packages.InstallRunParams{}
+	params := &packages.InstallRunParams{NamespaceType: &model.NamespaceBundle}
```

Comment:
Instead of passing namespace type to the runner as an argument, pass it in the params. We'll do this for languages and platforms too.